plugins/modules/mysql_user.py

```diff
@@ -975,6 +967,22 @@ def convert_priv_dict_to_str(priv):
     return '/'.join(priv_list)
 
 
+def handle_requiressl_in_priv_string(module, priv, tls_requires):
```

Comment:
```suggestion
def handle_requiressl_in_priv_string(module, priv, tls_requires):
```
@Jorge-Rodriguez i don't see where the function is invoked or I missed something

Reply:
Hah! Good one. It's invoked in my head, apparently. Let me fix that.